apps/docs/content/components/Image.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Image
+status: Experimental
+# figma: https://www.figma.com/file/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=539%3A14285&t=RJEWZhVUtrRQ0pKh-0
+source: https://github.com/primer/brand/blob/main/packages/react/src/Image/Image.tsx
+storybook: '/brand/storybook/?path=/docs/components-image--playground'
+description: Use Image to create both normal and responsive image elements.
+---
+
+import {Label} from '@primer/react'
+
+```js
+import {Image} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+```jsx live
+<Image src="https://source.unsplash.com/random" alt="Random Image" />
+```
+
+### Picture
+
+The `as` prop can be used to set the container of the image to be a picture component.
+
+```jsx live
+<Image
+  src="https://source.unsplash.com/random"
+  alt="Random Image"
+  as="picture"
+/>
+```
+
+### Aspect ratio
+
+The `aspectRatio` prop can be used to set the aspect ratio of the image. This is useful when the image is not the same aspect ratio as the container.
+
+```jsx live
+<Image
+  src="https://source.unsplash.com/random"
+  alt="Random Image"
+  aspectRatio="16:9"
+/>
+```
+
+### Height
+
+The `height` prop can be used to set the height of the image. This can be used along side the `aspectRatio` prop to create a responsive image the same size as other images.
+
+```jsx live
+<Image
+  src="https://source.unsplash.com/random"
+  alt="Random Image"
+  height={200}
+  aspectRatio="16:9"
+/>
+```
+
+### Width
+
+The `width` prop can be used to set the width of the image. This can be used along side the `aspectRatio` prop to create a responsive image the same size as other images.
+
+```jsx live
+<Image
+  src="https://source.unsplash.com/random"
+  alt="Random Image"
+  width={200}
+  aspectRatio="16:9"
```

Comment:
The aspect ratio here doesn't seem to be 16:9, is it working correctly? It looks more square than rectangular.

![Screenshot 2023-04-14 at 11 13 10](https://user-images.githubusercontent.com/13340707/232017218-35ee40f0-e728-4fea-bedb-ebc5a0ad555f.png)